exporter/clickhouseexporter/config.go

```diff
@@ -50,6 +52,21 @@ type Config struct {
 	// Ignored if async inserts are configured in the `endpoint` or `connection_params`.
 	// Async inserts may still be overridden server-side.
 	AsyncInsert bool `mapstructure:"async_insert"`
+	// MetricsTables defines the table names for metric types.
+	MetricsTables MetricTablesConfig `mapstructure:"metrics_tables"`
+}
+
+type MetricTablesConfig struct {
+	// Gauge is the table name for gauge metric type. default is `otel_metrics_gauge`.
+	Gauge internal.MetricTypeConfig `mapstructure:"gauge"`
```

Comment:
`Gauge internal.MetricTypeConfig` expected config: 
```yaml
gauge:
  name: "otel_metrics_gauge"
```
It looks like a redundant struct, so why not use: `Gauge String`

Reply:
it's for potential future configuration options, if it will be decided to extend the configuration, it will be very easy to implement it, since all the structs and logic are already prepared for it

Reply:
I think that's not possible, since we need to use that struct in the internal package, which will lead to the need to importing `clickhouseexporter` package there -> this will lead to cyclic dependencies